### PR TITLE
rm-improved: init at 0.13.0

### PR DESCRIPTION
**Pull request by nils-degroot**
###### Motivation for this change

Hi all. Previously I used this package on a non Nixos machine. The resulting binary (rip) is similar to rm but has a undo.
Here is the source: https://github.com/nivekuil/rip

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -7582,6 +7582,12 @@
     githubId = 26231126;
     name = "Nils ANDRÉ-CHANG";
   };
+  nils-degroot = {
+    email = "nils@peeko.nl";
+    github = "nils-degroot";
+    githubId = ;
```
Comment (lourkeur) [inline maintainers/maintainer-list.nix:7588]:
```suggestion
    githubId = 53556985;
```

per https://api.github.com/users/nils-degroot

File: pkgs/applications/misc/rm-improved/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    rev = "0.13.0";
+    sha256 = "0d065xia4mwdhxkiqfg7pic6scfzipzmsvvx7l6l97w62lzpiqx3";
+  };
+
+  meta = {
+    description = "rip is a command-line deletion tool focused on safety, ergonomics, and performance";
+    longDescription = "rip is a command-line deletion tool focused on safety, ergonomics, and performance. It favors a simple interface, and does not implement the xdg-trash spec or attempt to achieve the same goals.";
+    homepage = "https://github.com/nivekuil/rip";
+    maintainers = with maintainers; [ nils-degroot ];
+    license = licenses.gpl3;
```
Comment (lourkeur) [inline pkgs/applications/misc/rm-improved/default.nix:23]:
```suggestion
    license = licenses.gpl3Plus;
```
ambiguity is deprecated in this case. Per cargo metadata, that codebase is "or any later version".

File: pkgs/applications/misc/rm-improved/default.nix [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, rustPlatform, lib }:
```
Comment (lourkeur) [inline pkgs/applications/misc/rm-improved/default.nix:1]:
```suggestion
{ fetchFromGitHub, rustPlatform, lib }:
```
Per `nixpkgs-hammer`

**Review by bbjubjub2494 — CHANGES_REQUESTED** [review]
The derivation itself LGTM. Per [guidelines](https://nixos.org/manual/nixpkgs/stable/#:~:text=if%20you%20have%20commits%20pkg-name%3A%20oh%2C%20forgot%20to%20insert%20whitespace%3A%20squash%20commits%20in%20this%20case.%20use%20git%20rebase%20-i.), you should rebase this into 2 commits:
- maintainers: add nils-degroot
- rm-improved: init at 0.13.0

**Comment by roberth** [comment]
@primeos It seems like you've pushed unrelated commits to this PR's branch, or did I miss something?

**Comment by lourkeur** [comment]
> @primeos It seems like you've pushed unrelated commits to this PR's branch, or did I miss something?

primeos had no hand in this.

@nils-degroot your 2 commits are compliant, you just need to rebase and nuke the duplicate off-topic ones.

**Comment by nils-degroot** [comment]
Sorry for the inconveniance. Got it right this time.

**Review by bbjubjub2494 — APPROVED** [review]
Result of `nixpkgs-review pr 130641` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>rm-improved</li>
  </ul>
</details>

File: pkgs/applications/misc/rm-improved/default.nix [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+  src = fetchFromGitHub {
+    owner = "nivekuil";
+    repo = "rip";
+    rev = "0.13.0";
+    sha256 = "0d065xia4mwdhxkiqfg7pic6scfzipzmsvvx7l6l97w62lzpiqx3";
+  };
+
+  meta = {
+    description = "rip is a command-line deletion tool focused on safety, ergonomics, and performance";
```
Comment (roberth) [inline pkgs/applications/misc/rm-improved/default.nix:19]: This description doesn't quite follow the contributing guidelines. See the link in the PR description. Please also check `longDescription`.

**Comment by roberth** [comment]
> primeos had no hand in this.

GitHub presented the commits that way. Sorry, primeos!

**Comment by roberth** [comment]
Also builds on darwin; no need to wait for ofborg.
Thanks, and welcome :)

**Comment by nils-degroot** [comment]
Glad to help :smile:

**Pull request merged**